web/gridt/tests/integration/resources/test_movements.py

```diff
@@ -226,6 +125,18 @@ def test_post_successful(self):
             self.assertEqual(len(movement.current_users), 1)
             self.assertEqual(movement.current_users[0], user)
 
+    @patch("marshmallow.Schema.load", side_effect=ValidationError({"message": "Error."}))
+    def test_post_unsuccessful(self, func):
+        with self.app_context():
+            user = self.create_user()
+
+            resp = self.request_as_user(
+                self.users[0], "POST", "/movements", json={},
+            )
+
+            func.assert_called_with({})
+            self.assertEqual(resp.status_code, 400)
```

Comment:
I'm not sure if testing the error code is sufficient here. It feels more natural to also test the error message itself. Especially since you mock it, you can be sure that it is indeed the mocked error that is raised and not some other 400 error.